docs/postman/sample_requests/sample_interaction.json

```diff
@@ -25,7 +25,7 @@
           {
             "idType": "URI",
             "type": "GlobalReference",
-            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model",
+            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/10JF-1234-Jf14-PP22",
```

Comment:
remove /10JF-1234-Jf14-PP2
the semanticId is the same for each submodel instance.

Reply:
done